js/theme.js

```diff
@@ -187,13 +190,27 @@ function ThemeNav () {
         });
     };
 
-    nav.toggleCurrent = function (elem) {
+    nav.toggleExpanded = function (elem, forceExpanded) {
         var parent_li = elem.closest('li');
+        // parent_li.siblings('li.current').removeClass('current');
+        // parent_li.siblings().find('li.current').removeClass('current');
+        // parent_li.find('> ul li.current').removeClass('current');
```

Comment:
Why is this commented out?

Reply:
This code moved to `setCurrent()`, but would be better to remove. I'll update the code.